src/solvers/euler_solver.C

```diff
@@ -190,5 +194,232 @@ bool EulerSolver::_general_residual (bool request_jacobian,
   return jacobian_computed;
 }
 
+void EulerSolver::integrate_qoi_timestep()
+{
+  // We are using the trapezoidal rule to integrate each timestep
```

Comment:
Another spot to point out that we're using theta now.

Reply:
Yup, got this.